docker/Dockerfile.release

```diff
@@ -11,7 +11,6 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 #
-# Download artifact from bintray, where the artifact should have been published by now.
```

Comment:
accident of license plugin, but incidentally it is redundant to a below comment :)